public/app/core/history/richHistoryStorageProvider.ts

```diff
@@ -0,0 +1,13 @@
+import RichHistoryLocalStorage from './richHistoryLocalStorage';
+import RichHistoryRemoteStorage from './richHistoryRemoteStorage';
+import RichHistoryStorage from './richHistoryStorage';
+
+const richHistoryLocalStorage = new RichHistoryLocalStorage();
+
+// To be implemented and switched with a feature flag.
+// @ts-ignore
+const richHistoryRemoteStorage = new RichHistoryRemoteStorage();
```

Comment:
can't we remove this since it's not being used?

Reply:
This will be used for backend data source. I just wanted to make it clear that there's gonna be a separate implementation soon. But I can remove it and add back only when it's actually used.

Reply:
I've removed it in c068ff78fcfc0bc5f7779e575c8797482e9fcdb9. I'll add it back when it's implemented.